vizro-ai/docs/pages/explanation/safety-in-vizro-ai.md

```diff
@@ -6,39 +6,44 @@ We recommend users research and understand the selected model before using `vizr
 
 Users are encouraged to treat AI-generated content as supplementary, **always apply human judgment**,
 approach with caution, review the relevant [disclaimer](disclaimer.md) page, and consider the following:
-
-### 1. Hallucination and misrepresentation
 <!-- vale off -->
+### 1. Hallucination and misrepresentation
+
 Generative models can potentially generate information while appearing factual, being entirely fictitious or misleading.
 <!-- vale on -->
 The vendor models might lack real-time knowledge or events beyond its last updates.
 `vizro_ai` output may vary and you should always verify critical information.
 It is the user's responsibility to discern the accuracy, consistent, and reliability of the generated content.
 
+<!-- vale off -->
```

Comment:
Will this be turned on later again?

Reply:
@maxschulz-COL I'm turning it off in comments when the rule is generally a good one (e.g. to enforce correct capitalisation in headings) but when it fails for a valid reason and I can't work out how to work around it my editing the style. In this case, it doesn't like numbered headings, so I have excluded them from checking. My skills with regex mean I've not work out how to do the exception in the style!